File: Bab 8/8.4 Repeat-Do While.java
import java.util.Scanner;

public class Main{

	public static void main(String[] args) {
		

		char jawaban;

		Scanner sc = new Scanner(System.in);

		do{
		
			System.out.println("Halo dunia!");

			System.out.println("Apakah anda ingin menampilkan lagi?");

			try{

				jawaban = sc.next().charAt(0);

			}catch(Exception e){

			}

		} while (jawaban == 'y');
	}
}

File: Bab 8/8.4 Repeat Do-While.c
/*
 *  Name Program    : Repeat Do-While
 *  Purpose         : Try Code from Logic Algorithm and Basic Programming Book chapter 8
 *  Author          : Yogi Siswanto
 *  Email           : yogisiswanto.c2@gmail.com
 *  Date Created    : 22th April 2021
 *  Language        : C Language
 */

#include <stdio.h>

// Main function
int main(){
	
	// Variable declaration
	char jawaban;

	// Getting input from console first, and then check
	do{
		
		printf("Halo dunia!\n");

		printf("Apakah anda ingin menampilkan lagi?\n");

		scanf("%c", &jawaban);

		while(getchar() != '\n');

		printf("\n");

	} while (jawaban == 'y');

	return 0;   
}